pkg/reconciler/knativeserving/knativeserving.go

```diff
@@ -74,7 +77,7 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, original *servingv1alpha1
 		}
 	}
 
-	manifest, err := r.transform(ctx, original)
+	manifest, err := r.getCurrentManifest(ctx, original)
 	if err != nil {
 		return fmt.Errorf("failed to transform manifest: %w", err)
```

Comment:
I think I'd just return err here, as "transform" isn't exactly accurate, as long as you ensure proper errors are returned from `getCurrentManifest`

Reply:
done.